### DISCUSSION TITLE
LXC on proxmox. NFS folder mounted from nas > /photorism/originals. Photoprism says originals empty

### DISCUSSION THREAD

**Discussion by Valkdd** [photoprism/photoprism#3269] [Q&A] [answered]
I've used this magnificent [link](https://tteck.github.io/Proxmox/) and installed photoprism in LXC, I want to use my existing photo library mounted on NAS as originals. I am mounting in fstab:
 NAS:/volume1/photo /photoprism/originals nfs defaults 0 0
I have access to the mounted folder from LXC console under root creds in /photoprism/originals and I do see all my photofolders there.
But photoprism says originals folder is empty when I try to scan library. So what am I missing here?

**Comment by lastzero** [comment]
I have no experience with LXC and Proxmox, but fstab is usually not the right file to mount directories in PhotoPrism (unless it is a completely custom solution).

**Comment by Valkdd** [comment, marked as answer]
Ok I am gonna answer this myself incase someone finds themselves in my shoes. So the correct path that this instance of photoprism considers originals is:
/var/lib/photoprism/photos/Originals/
so fstab line should look like this:
NAS:/volume1/photo /var/lib/photoprism/photos/Originals/ nfs defaults 0 0

**Reply by mresetar-croz** [reply]
In the latest PVE 8 location is at `/opt/photoprism/photos/originals/ `

My mount point in `vi /etc/pve/lxc/100.conf` looks like 
`mp0: /mnt/pictures,mp=/opt/photoprism/photos/originals`

**Reply by lastzero** [reply]
Be careful when you store your photos in /opt/photoprism as its contents are removed and recreated if you follow our docs for manual version upgrades: https://dl.photoprism.app/pkg/linux/README.html

If possible, it's better to use e.g. /photoprism instead as this way you don't mix distribution files with your own files.

**Reply by mresetar-croz** [reply]
Thanks for the heads up. It was weird that `/opt/photoprism/config/.env` had defaults storage in `/opt/photoprism`. 
Probably there is a reason why, I just rolled with it. I would change this going further.  

```bash
PHOTOPRISM_STORAGE_PATH='/opt/photoprism/storage'
PHOTOPRISM_ORIGINALS_PATH='/opt/photoprism/photos/originals'
PHOTOPRISM_IMPORT_PATH='/opt/photoprism/photos/import'
```

**Reply by lastzero** [reply]
We've updated our related docs to clarify that **we do not recommend mixing custom and distribution files** in `/opt/photoprism`, since this makes updates much more complicated and can lead to data loss if you follow the standard procedure of replacing the contents of the `/opt/photoprism` directory:

- https://www.photoprism.app/kb/installation-packages#updates
- https://www.photoprism.app/kb/installation-packages#configuration

**It would be great if you could let the maintainers of these scripts/integrations know about this!**
